package/gluon-setup-mode/luasrc/lib/gluon/upgrade/350-outdoor-preset

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/lua
```

Comment:
This is the script that detects outdoor models and sets the preset. It will run unless the outdoor variable was already discovered. 

If autodiscovery fails debug here.

Reply:
@Tata Discovery will only happen on first boot of device afair.